src/coldcard.js

```diff
@@ -593,8 +609,6 @@ export class ColdcardMultisigWalletConfig {
    * @returns {string} output to be written to a text file and uploaded to Coldcard.
    */
   adapt() {
-    // Coldcard configs can't have spaces in the names, it just splits on space and takes the first word.
```

Comment:
Is this no longer true about about names?

Reply:
Ya - turns out it wasn't spaces. the problem was that our default names were e.g. `Vault #1` and then the # would cause the line to be ignored making it seem like the space was the problem.

See https://github.com/unchained-capital/unchained-wallets/issues/48 for more details

Reply:
Yeah, they fixed that in the latest firmware version